lib/stdlib/src/proplists.erl

```diff
@@ -39,6 +39,7 @@
 %% etc.</p>
 %%
 %% % @type property() = atom() | tuple()
+%% % @type proplist() = [property()]
```

Comment:
We plan to remove all the Edoc documentation from proplists.erl since it is redundant and inconsistent. There are already type , spec etc in the syntax and the documentation which we actually publish is in the doc/src(/proplists.xml. So an older version of the same doc in the .erl file makes no sense in this case.